src/VisualStudio/Core/Test/FindResults/FindResultsTests.vb

```diff
@@ -0,0 +1,179 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Text
+Imports System.Threading
+Imports Microsoft.CodeAnalysis
+Imports Microsoft.CodeAnalysis.Editor.UnitTests
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.Extensions
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.Utilities
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces
+Imports Microsoft.CodeAnalysis.FindSymbols
+Imports Microsoft.VisualStudio.Composition
+Imports Microsoft.VisualStudio.LanguageServices.Implementation.Library.FindResults
+Imports Microsoft.VisualStudio.LanguageServices.UnitTests.ObjectBrowser.Mocks
+Imports Roslyn.Test.Utilities
+Imports Roslyn.Utilities
+
+Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.FindResults
+    Public Class FindResultsTests
+        <Fact, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        <WorkItem(1138943)>
+        Public Sub ConstructorReferencesShouldNotAppearUnderClassNodeInCSharp()
+            Dim markup = <Text><![CDATA[
+class $$C
+{
+    const int z = 1;
+
+    public C() { }
+    public C(int x) { }
+
+    void T()
+    {
+        var a = new C();
+        var b = new C(5);
+        var c = C.z;
+    }
+}"]]></Text>
+
+            Dim expectedResults = New List(Of AbstractTreeItem) From
+                {
+                    New TestFindResult("[CSharpAssembly1] C.C() (1 reference)",
```

Comment:
I even thought about this when writing the tests, and I was so going to remember to do it and you were going to be so proud and everything was going to be glorious. But here we are...